docs/grid-computing/grid-universe.rst

```diff
@@ -354,62 +357,111 @@ The following table summarizes how job classad attributes will be translated int
+the normal SSH connection method can be made to work.
+
+For the Reverse GAHP to work, your local machine must be reachable on
+the network from the remote machine on the SSH and HTCondor ports
+(22 and 9618, respectively).
+Also, your local machine must allow SSH logins using just an ssh key
+for authentication.
+
+First, run the *condor_remote_cluster* as you would for a regular
+remote SSH setup.
 
 .. code-block:: text
 
-    grid_resource = lsf
+    condor_remote_cluster --add alice@login.example.edu slurm
 
-:index:`SGE (Sun Grid Engine)`
-:index:`submitting jobs to SGE<single: submitting jobs to SGE; grid computing>`
+Second, create an ssh key that's authorized to login to your account on
+your local machine and save the private key on the remote machine.
+The private key should not be protected with a passphrase.
+In the following examples, we'll assume the ssh private key is named
+``~/.ssh/id_rsa_rvgahp``.
 
-The popular Grid Engine batch system (formerly known as Sun Grid Engine
-and abbreviated SGE) is available in two varieties: Oracle Grid Engine
-(`http://www.oracle.com/us/products/tools/oracle-grid-engine-075549.html <http://www.oracle.com/us/products/tools/oracle-grid-engine-075549.html>`_)
-and Univa Grid Engine
-(`http://www.univa.com/?gclid=CLXg6-OEy6wCFWICQAodl0lm9Q <http://www.univa.com/?gclid=CLXg6-OEy6wCFWICQAodl0lm9Q>`_).
+Third, select a pathname on your local machine for a unix socket file
+that will be used by the Reverse GAHP components to communicate with
+each other.
+The Reverse GAHP programs will create the file as your user identity,
+so we suggest using a location under your home directory or /tmp.
+In the following examples, we'll use ``/tmp/alice.rvgahp.socket``.
 
-As an alternative to the submission details given above, HTCondor jobs
-may be submitted to a local SGE system using the **grid** universe and
-adding the **grid_resource** command by placing into the submit
-description file:
+Fourth, on the remote machine, create a ``~/bosco/glite/bin/rvgahp_ssh``
+shell script like this:
 
 .. code-block:: text
 
-    grid_resource = sge
+    #!/bin/bash
+    exec ssh -o "ServerAliveInterval 60" -o "BatchMode yes" -i ~/.ssh/id_rsa_rvgahp alice@submithost "/usr/sbin/rvgahp_proxy /tmp/alice.rvgahp.sock"
 
-The *condor_qsub* command line tool will take PBS/SGE style batch files
-or command line arguments and submit the job to HTCondor instead. See
-the :doc:`/man-pages/condor_qsub` manual page for details.
+Run this script manually to ensure it works.
+It should print a couple messages from the *rvgahp_proxy* started on your
+local machine.
+You can kill the program once it's working correctly.
+
+.. code-block:: text
+
+    2022-03-23 13:06:08.304520 rvgahp_proxy[8169]: rvgahp_proxy starting...
+    2022-03-23 13:06:08.304766 rvgahp_proxy[8169]: UNIX socket: /tmp/alice.rvgahp.sock
+
+Fifth, run the *rvgahp_server* program on the remote machine.
```

Comment:
I think that it might be nice to end this list of steps with 'Finally'. Let me know if you agree.
```suggestion
Finally, run the *rvgahp_server* program on the remote machine.
```